xml/vk.xml

```diff
@@ -34,6 +34,7 @@ branch of the member gitlab server.
         <platform name="ggp" protect="VK_USE_PLATFORM_GGP" comment="Google Games Platform"/>
         <platform name="provisional" protect="VK_ENABLE_BETA_EXTENSIONS" comment="Enable declarations for beta/provisional extensions"/>
         <platform name="screen" protect="VK_USE_PLATFORM_SCREEN_QNX" comment="QNX Screen Graphics Subsystem"/>
+        <platform name="drm" protect="VK_USE_PLATFORM_DRM_KHR" comment="Direct Rendering Manager interface"/>
```

Comment:
Do we need a separate `vulkan_drm.h` header here? The extension doesn't rely on any platform-specific header.

Reply:
Shoud I put it in the main header, inside linux specific defines?

Reply:
Hm, I'm not sure I understand. I don't think we need to have `VK_USE_PLATFORM_DRM_KHR` nor `vulkan_drm.h`.

See https://github.com/KhronosGroup/Vulkan-Docs/pull/1356 for an example of an ext that is DRM-specific but doesn't need to pull any platform-specific header.

Reply:
`VK_EXT_acquire_xlib_display` has its own header, `VK_KHR_xlib_surface` and `VK_KHR_wayland_surface` too. This extension being Linux only, but not tied to a specific "platform", I was hesitating between adding a `linux` or a `drm` specific header.

Reply:
Why do we need to specify a `platform` at all?

Reply:
Should the extension be Linux specific?

Reply:
Probably not, it should work on BSDs too.

Reply:
So I can leave it in the main header, and it should be unimplemented on non-unix platforms?

Reply:
I ended up following what `VK_EXT_image_drm_format_modifier` did.